src/collective/solr/testing.py

```diff
@@ -194,11 +198,13 @@ def activateAndReindex(portal):
 COLLECTIVE_SOLR_FIXTURE = CollectiveSolrLayer()
 
 COLLECTIVE_SOLR_INTEGRATION_TESTING = IntegrationTesting(
-    bases=(LEGACY_COLLECTIVE_SOLR_FIXTURE,),
+    bases=(PLONE_APP_REGISTRY_FIXTURE,
```

Comment:
I am commenting from far away, but try to run the tests with plone.testing 5.0.0 They might produce more testfailures because of what tisto saw.